docs/hacking.md

```diff
@@ -243,7 +243,7 @@ spec:
   - openshift-migration
 EOF
 
-oc apply -f operatorgroup.yml
+oc create -f operatorgroup.yml
```

Comment:
Since this is using `generateName`, we should use `oc create` instead of `oc apply`